addrmgr/addrmanager_test.go

```diff
@@ -910,148 +913,159 @@ func TestValidatePeerNa(t *testing.T) {
+		}, {
+			name:          "routable ipv6 to routable ipv4",
+			localAddr:     routableIPv6Addr,
+			remoteAddr:    routableIPv4Addr,
+			expectedBool:  false,
+			expectedReach: Unreachable,
+		}, {
+			name:          "non-routable ipv4 to routable ipv6",
+			localAddr:     nonRoutableIPv4Addr,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  false,
+			expectedReach: Default,
+		}, {
+			name:          "non-routable ipv6 to routable ipv6",
+			localAddr:     nonRoutableIPv6Addr,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  false,
+			expectedReach: Default,
+		}, {
+			name:          "non-routable ipv4 to routable ipv6",
+			localAddr:     nonRoutableIPv4Addr,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  false,
+			expectedReach: Default,
+		}, {
+			name:          "routable ipv4 to non-routable ipv6",
+			localAddr:     routableIPv4Addr,
+			remoteAddr:    nonRoutableIPv6Addr,
+			expectedBool:  false,
+			expectedReach: Unreachable,
+		}, {
+			name:          "routable ivp6 rfc4380 to routable ipv6",
+			localAddr:     rfc4380IPAddress,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  true,
+			expectedReach: Teredo,
+		}, {
+			name:          "routable ipv4 to routable ipv6",
+			localAddr:     routableIPv4Addr,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  true,
+			expectedReach: Ipv4,
+		}, {
+			name:          "tunnelled ipv6 rfc3964 to routable ipv6",
+			localAddr:     rfc3964IPAddress,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  true,
+			expectedReach: Ipv6Weak,
+		}, {
+			name:          "tunnelled ipv6 rfc6052 to routable ipv6",
+			localAddr:     rfc6052IPAddress,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  true,
+			expectedReach: Ipv6Weak,
+		}, {
+			name:          "tunnelled ipv6 rfc6145 to routable ipv6",
+			localAddr:     rfc6145IPAddress,
+			remoteAddr:    routableIPv6Addr,
+			expectedBool:  true,
+			expectedReach: Ipv6Weak,
+		},
+	}
 
-	addressManager := New("testValidatePeerNa", nil)
+	addressManager := New("TestIsExternalAddrCandidate", nil)
 	for _, test := range tests {
-		localIP := net.ParseIP(test.localAddress)
-		remoteIP := net.ParseIP(test.remoteAddress)
-		localNa := NewNetAddressIPPort(localIP, 8333, wire.SFNodeNetwork)
-		remoteNa := NewNetAddressIPPort(remoteIP, 8333, wire.SFNodeNetwork)
-
-		valid, reach := addressManager.ValidatePeerNa(localNa, remoteNa)
-		if valid != test.valid {
-			t.Errorf("%q: unexpected return value for valid - want '%v', "+
-				"got '%v'", test.name, test.valid, valid)
+		localIP := net.ParseIP(test.localAddr)
+		remoteIP := net.ParseIP(test.remoteAddr)
+		localNa := NewNetAddressFromIPPort(localIP, 8333, wire.SFNodeNetwork)
+		remoteNa := NewNetAddressFromIPPort(remoteIP, 8333, wire.SFNodeNetwork)
+
+		goodReach, reach := addressManager.IsExternalAddrCandidate(localNa, remoteNa)
+		if goodReach != test.expectedBool {
+			t.Errorf("%q: unexpected return value for bool - expected '%v', "+
```

Comment:
Totally trivial, but virtual every test in the entire code base use `want` and `got`, so changing to `expected` is quite inconsistent.

Reply:
Updated.